packages/ui/src/components/CookieBanner/CookieBanner.tsx

```diff
@@ -32,32 +31,25 @@ export const CookieBanner = (props: CookieBannerProps) => {
       bg="gray.700"
       rounded={'md'}
       boxShadow="sm"
-      {...rest}
+      position="fixed"
+      bottom="0"
+      right="0"
+      left="0"
```

Comment:
If it's rounded, I prefer to have some space around the banner

```suggestion
      bottom="2"
      right="2"
      left="2"
```